NitroxPatcher/Patches/BaseGhost_Finish_Patch.cs

```diff
@@ -32,7 +32,7 @@ public static IEnumerable<CodeInstruction> Transpiler(MethodBase original, IEnum
                      * TransientLocalObjectManager.Add(TransientLocalObjectManager.TransientObjectType.BASE_GHOST_NEWLY_CONSTRUCTED_BASE_GAMEOBJECT, gameObject);
                      */
                     yield return new CodeInstruction(OpCodes.Ldc_I4_1);
-                    yield return original.Ldloc<GameObject>(1);
+                    yield return original?.Ldloc<GameObject>(1);
```

Comment:
These should not be nullable.  We need an exception or they will silently fail if UWE changes them.

Reply:
Made it that way due too the unitTest passing in a null parameter...